boards/riscv/tlsr9518adk80d/Kconfig.defconfig

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2021 Telink Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD
```

Comment:
It looks like the whole file should be inside an `if BOARD_TLSR9518ADK80D` instead.

Reply:
Done